Amasty_Storelocator/styles/modules/_store-locator.scss

```diff
@@ -147,6 +169,88 @@
     }
 }
 
+.amlocator-location-container {
+    .amlocator-modes {
+        border: none;
+    }
+
+    .amlocator-column .amlocator-button {
+        color: $white;
```

Comment:
this color should go to variable :) this white is for buttons, so it can goes to one variable